README.md

```diff
@@ -24,97 +21,52 @@ To use this module in a new project, create a file such as:
-
-  # version = "0.1.0"
-
-  # TEMPLATE: insert required variables here
+module "nginx-single" {
+  source                 = "equinix-labs/network-edge-device-nginx/equinix"
+  name                   = "terraform-test-NGINX"
+  hostname               = "terraform-nginx"
+  metro_code             = var.metro_code_primary
+  account_number         = "123456"
+  platform               = "small"
+  software_package       = "STD"
+  term_length            = 1
+  notifications          = ["test@test.com"]
+  additional_bandwidth   = 50
+  mgmt_acl_template_uuid = equinix_network_acl_template.nginx-pri.id
+  ssh_key = {
+    userName = "johndoe"
+    keyName  = equinix_network_ssh_key.johndoe.name
+  }
 }
-```
-
-Install [pre-commit](https://pre-commit.com/#install) with its prerequesites: [python](https://docs.python.org/3/using/index.html) and [pip](https://pip.pypa.io/en/stable/installation/).
-
-Configure pre-commit: `pre-commit install`.
 
-Install required packages: [tflint](https://github.com/terraform-linters/tflint), [tfsec](https://aquasecurity.github.io/tfsec/v1.0.11/getting-started/installation/), [shfmt](https://github.com/mvdan/sh), [shellcheck](https://github.com/koalaman/shellcheck), and [markdownlint](https://github.com/markdownlint/markdownlint).
+```
 
 Run `terraform init -upgrade` and `terraform apply`.
 
-## Module Documentation
-
-The main README.md, the modules README.md and the examples README.md are populated by [terraform-docs worflow job](.github/workflows/documentation.yaml). The following sections are appended between the terraform-docs delimeters: Requiremenents, Providers, Modules, Resources, Inputs, and Outputs.
-
-## Module Release and Changelog Generation
-
-The module git release and [changelog](CHANGELOG.md) are generated by the [release workflow job](.github/workflows/release.yaml). The release worflow follows the [conventional commits convention](https://www.conventionalcommits.org/). To submit a commit, please follow the [commit message format guidelines](https://www.conventionalcommits.org/en/v1.0.0/#specification). This job is set to run manually by default.
-
-Example commit message: `fix: disabled log generation for system services`
+### Variables
 
-For more examples, please see [conventional commit message examples](https://www.conventionalcommits.org/en/v1.0.0/#examples).
+See <https://registry.terraform.io/modules/equinix-labs/network-edge-device-nginx/equinix/latest?tab=inputs> for a description of all variables.
 
-## Examples
+### Outputs
 
-To view examples for how you can leverage this module, please see the [examples](examples/) directory.
+See <https://registry.terraform.io/modules/equinix-labs/network-edge-device-nginx/equinix/latest?tab=outputs> for a description of all outputs.
 
-<!-- TEMPLATE: The following block has been generated by terraform-docs util: https://github.com/terraform-docs/terraform-docs -->
-<!-- BEGIN_TF_DOCS -->
```

Comment:
@schadalawada The requirements, variables, and outputs of this module can be automatically maintained by workflows that come included in `.github`. You'll need to keep the `<!-- BEGIN_TF_DOCS -->` section to keep that behavior (recommended).

Reply:
@schadalawada I saw you added the comment back. We have a GH action to generate documentation that uses that block. It will generate next sections for you: Requirements, Providers, Modules, Resources, Inputs, Outputs. 

Please remove all those sections in the README file and just add this code where you want them

```
<!-- BEGIN_TF_DOCS -->
<!-- END_TF_DOCS -->
```

Reply:
@schadalawada added BEGIN_TF_DOCS blocks. These are not in all the correct places, some custom content will be overridden by CI and some of the custom content blocks (in examples) is missing BEGIN_TF_DOCS. I'm going to resolve this thread so we can merge this. We'll want to chase this with corrections to the location of BEGIN_TF_DOCS blocks.